Fatal error while trying to install Mastodon on Raspberry Pi 4

Note: This is a copy of : https://github.com/YunoHost/issues/issues/1668

I attempted to install Mastodon on my Yunohost and got a fatal error. I'm reporting the error here as the app suggested me to do.

Screenshot of the error:
![](https://i.imgur.com/Sde0TVs.png)

Paste of the error:
https://paste.yunohost.org/raw/yewivizido

My device:
- Raspberry Pi 4
- 4GB of Ram
- Freshly installed Yunohost last week
- Other installed apps on the same device: Bibliogram, RoundCube, Discourse

I'm very new to Yunohost but I find it very cool so far. Don't hesitate to ask for more infos. I'll help as much as I can.

Indeed, these days many people report that same issue about bcrypt failing to compile

```
2020-10-20 00:21:10,435: WARNING - current directory:
2020-10-20 00:21:10,436: WARNING - /var/www/mastodon/live/vendor/bundle/ruby/2.6.0/gems/bcrypt-3.1.13/ext/mri
2020-10-20 00:21:10,437: WARNING - make "DESTDIR="
2020-10-20 00:21:10,439: WARNING - compiling bcrypt_ext.c
2020-10-20 00:21:10,440: WARNING - compiling crypt_blowfish.c
2020-10-20 00:21:10,441: WARNING - gcc  -D__SKIP_GNU -I/opt/rbenv/versions/2.6.6/include  -D_FILE_OFFSET_BITS=64
2020-10-20 00:21:10,442: WARNING - -c -o x86.o x86.S
2020-10-20 00:21:10,444: WARNING - x86.S: Assembler messages:
2020-10-20 00:21:10,445: WARNING - x86.S:202: Error: junk at end of line, first unrecognized character is `,'
2020-10-20 00:21:10,446: WARNING - make: *** [<builtin>: x86.o] Error 1
[...]
2020-10-20 00:21:10,458: WARNING - An error occurred while installing bcrypt (3.1.13), and Bundler cannot continue.
2020-10-20 00:21:10,459: WARNING - Make sure that `gem install bcrypt -v '3.1.13' --source 'https://rubygems.org/'`
2020-10-20 00:21:10,460: WARNING - succeeds before bundling.
```

I'm really not knowledgeable enough about this app and ruby to know how to fix it tho, maybe the people maintaining the app know better :/

--- a/README.md
+++ b/README.md
@@ -11,7 +11,7 @@ If you don't have YunoHost, please see [here](https://yunohost.org/#/install) to
 ## Overview
 Mastodon is a free, open-source microblogging social network. It is a decentralized alternative to commercial platforms like Twitter and avoids the risks of a single company monopolizing your communication for commercial purposes. 
 
-**Shipped version:** 3.2.1
+**Shipped version:** 3.2.2
 
 ## Important points to read before installing
 

--- a/README_fr.md
+++ b/README_fr.md
@@ -11,7 +11,7 @@ Si vous n'avez pas YunoHost, regardez [ici](https://yunohost.org/#/install) pour
 ## Vue d'ensemble
 Mastodon est un réseau social de microblog auto-hébergé et open source. C'est une alternative décentralisée aux plates-formes commerciales comme Twitter. Mastodon évite ainsi les risques qu'une seule société monopolise votre communication à des fins commerciales.
 
-**Version incluse:** 3.2.1
+**Version incluse:** 3.2.2
 
 ## Points importants à lire avant l'installation
 

--- a/check_process
+++ b/check_process
@@ -23,6 +23,8 @@
 		upgrade=1	from_commit=bde14cc4496445fdfed4616ac44fc9527c8369fd
 		# 3.2.1~ynh1
 		upgrade=1	from_commit=248b717f373f3c82bdb19c33cf20980494608697
+		# 3.2.1~ynh2
+		upgrade=1	from_commit=74908bf3e70c134bc838c7db3ae8b5aa0938cd75
 		backup_restore=1
 		multi_instance=1
 		# This test is no longer necessary since the version 2.7 (PR: https://github.com/YunoHost/yunohost/pull/304), you can still do it if your app could be installed with this version.
@@ -46,3 +48,5 @@ Notification=all
 		name=3.2.0~ynh1
 	; commit=248b717f373f3c82bdb19c33cf20980494608697
 		name=3.2.1~ynh1
+	; commit=74908bf3e70c134bc838c7db3ae8b5aa0938cd75
+		name=3.2.1~ynh2

--- a/conf/app.src
+++ b/conf/app.src
@@ -1,5 +1,5 @@
-SOURCE_URL=https://github.com/tootsuite/mastodon/archive/v3.2.1.tar.gz
-SOURCE_SUM=429dbe1166948e2863dd97b7d5458d18887e7a75b7ff5005cfa50828f9c7b9e2 
+SOURCE_URL=https://github.com/tootsuite/mastodon/archive/v3.2.2.tar.gz
+SOURCE_SUM=298396b1bd0065689c3d9225c8466c6651979834acb4eec511057ff1ba630781 
 SOURCE_SUM_PRG=sha256sum
 SOURCE_FORMAT=tar.gz
 SOURCE_IN_SUBDIR=true

--- a/manifest.json
+++ b/manifest.json
@@ -6,7 +6,7 @@
         "en": "A libre and federated social network.",
         "fr": "Un réseau social libre et fédéré."
     },
-    "version": "3.2.1~ynh2",
+    "version": "3.2.2~ynh1",
     "url": "https://github.com/tootsuite/mastodon",
     "license": "AGPL-3.0-or-later",
     "maintainer": [

